article/_posts/2019-05-30-data-fusion-remote-sensing.md

```diff
@@ -1,23 +1,21 @@
 ---
 layout: review
-title:  "Multisource and Multitemporal Data Fusion in Remote Sensing"
-tags:   deep-learning segmentation remote-sensing CNN hyperspectral multi-task-learning point-cloud
-author: Charles Authier
-pdf:  https://arxiv.org/pdf/1812.08287.pdf
+title:  Multisource and Multitemporal Data Fusion in Remote Sensing
+tags:   deep-learning segmentation remote-sensing CNN hyperspectral multi-task-learning point-cloudr
```

Comment:
This should have not passed the pre-commit hook since *point-couldr* does not exist as a keyword. Please, configure the hooks by following the instructions in the  [how-to](https://github.com/vitalab/vitalab.github.io/blob/master/how-to.md) page:
```
2. Set up the `pre-commit` hooks by executing the `utils/setup_hooks.sh` script.
```

Reply:
Sorry, I meen `point-cloud`